launcher/internal/experiments/experiments.go

```diff
@@ -11,10 +11,11 @@ import (
 // Failure to unmarshal the experiment JSON data will result in an empty object being returned
 // to treat experiment flags as their default value. The error should still be checked.
 type Experiments struct {
-	EnableTestFeatureForImage  bool
-	EnableSignedContainerCache bool
-	EnableMeasureMemoryMonitor bool
-	EnableTempFSMount          bool
+	EnableTestFeatureForImage   bool
+	EnableSignedContainerCache  bool
+	EnableMeasureMemoryMonitor  bool
+	EnableTempFSMount           bool
+	EnableGpuDriverInstallation bool
```

Comment:
Have you try to allowlist the test project on the experiment? otherwise the cloudbuild test might not pass

Reply:
yes, test project is in the allowlist.